docker-compose.override.yaml

```diff
@@ -9,7 +9,7 @@ services:
       context: .
       target: development
     image: ffc-demo-payment-service
-    command: sh -c "scripts/wait-for-dependencies && npm run start:watch"
+    command: sh -c "npm run start:watch"
```

Comment:
This is already the command in the Dockerfile so no need for any command override.